test/e2e/minimatch/minimatch_setup.go

```diff
@@ -45,6 +44,17 @@ func createMinimatchForTest(t *testing.T) *rpcTesting.TestContext {
 		})
 		assert.Nil(t, minimatch.BindService(p, cfg))
 	})
+	// TODO: Revisit the Minimatch test setup in future milestone to simplify passing config
```

Comment:
FYI: I have a few changes that's going to change Minimatch coming this week. One is already in review.

Reply:
Sounds good. I ve another one in the works again adding some tweaks to mm. We ll discuss on that review if thats needed. Thanks!